diff -Nru ganso-0.1.1/codecs/video/ucbmpeg/Makefile.am ganso-0.1.1.new/codecs/video/ucbmpeg/Makefile.am
--- ganso-0.1.1/codecs/video/ucbmpeg/Makefile.am	Mon Aug 21 01:00:02 2000
+++ ganso-0.1.1.new/codecs/video/ucbmpeg/Makefile.am	Sat Mar  3 15:17:04 2001
@@ -2,7 +2,7 @@
         $(GNOME_INCLUDEDIR)
 
 ucbmpeg_la_LIBADD = -lm
-ucbmpeg_la_LDFLAGS = $(extra_libraries) -module -no-undefined -avoid-version
+ucbmpeg_la_LDFLAGS = -module -no-undefined -avoid-version
 
 sublib_LTLIBRARIES = ucbmpeg.la
 
diff -Nru ganso-0.1.1/plugins/sample/Makefile.am ganso-0.1.1.new/plugins/sample/Makefile.am
--- ganso-0.1.1/plugins/sample/Makefile.am	Mon Aug 21 01:00:02 2000
+++ ganso-0.1.1.new/plugins/sample/Makefile.am	Sat Mar  3 15:19:09 2001
@@ -2,7 +2,7 @@
         $(GNOME_INCLUDEDIR) $(XML_CFLAGS)
 
 sample_la_LIBADD = -lm
-sample_la_LDFLAGS = $(extra_libraries) -module -no-undefined -avoid-version
+sample_la_LDFLAGS = -module -no-undefined -avoid-version
 
 sublib_LTLIBRARIES = sample.la
 
